App/res/layout/guide_conclusion.xml

```diff
@@ -20,4 +20,14 @@
         android:textColor="@color/white"
         android:text="@string/complete_this_guide"/>
 
+    <Button
+        android:id="@+id/didnt_complete_guide"
+        android:visibility="invisible"
```

Comment:
Should this be initialized to `gone` instead?

Also, would it be easier to reuse the same button and merely change the text and click action rather than showing/hiding two separate buttons?

Reply:
Well ,  I have some suggestions if anyone would like to know....